### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by 2bleO**
In this PR:

- Create class MyList.
- Create each method inside the class.
- Create module MyEnumerable used inside MyList class.
- Create all?, any? and filter? methods in the MyEnumerable module.

### CODE REVIEW

File: my_list.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+class MyList
```
Comment (sinansevgi) [inline my_list.rb:3]: - [x] You should import your Enumerable methods to this file in order to use them.

```suggestion
require_relative './my_enumerable.rb'

class MyList
```

File: README.md [hunk 2]
```diff
@@ -1 +1,26 @@
-# enumerable
\ No newline at end of file
+# enumerable
+Replicate enumerables any?, all? and filter.
+
+# Built with
+- Ruby
+
```
Comment (sinansevgi) [inline README.md:6]: - [x] Documenting your code is one of the core skills you need to learn as a junior developer. And The microverse projects are a really great place to test your skills and get feedback on them. I recommend you document all your projects properly. Please add the necessary directives to run and use the project.

**Review by sinansevgi — CHANGES_REQUESTED** [review]
# Changes Required :recycle: 
------

Hi @2bleO , 

Good job so far!
All methods work well :+1: 
There are some issues that you still need to work on to go further but you are almost there!


 ## Required Changes ♻️
_Please check the inline comments for my suggestions_ :arrow_down:

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi

------

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by oxenprogrammer — APPROVED** [review]
Hi @2bleO, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**